index.markdown

```diff
@@ -4,28 +4,65 @@ title: Digital Tools for Trade Unions
 
-[![The Communication Workers Union]({{ site.url }}{{ site.baseurl }}/assets/images/cwu-logo.svg)](https://www.cwu.org) 
-[![Bakers Food and Allied Workers Union]({{ site.url }}{{ site.baseurl }}/assets/images/bfawu-logo.png)](https://www.bfawu.org) 
-[![Writers Guild of Great Britain]({{ site.url }}{{ site.baseurl }}/assets/images/wggb-logo.jpg)](https://writersguild.org.uk) 
-[![Forsa the Irish Trade Union]({{ site.url }}{{ site.baseurl }}/assets/images/forsa-logo.png)](https://www.forsa.ie) 
-[![Trade Union Congress]({{ site.url }}{{ site.baseurl }}/assets/images/tuc-logo.jpg)](https://www.tuc.org.uk) 
+## Online join that is optimised to get you more members
+
+We sweat the details so that your online join gets you more members: 60% of applicants that start a Join Together join form go on to submit their application. How many extra members would your union get with a form that converted that well?
+
+## Bespoke forms that handle all the complexities of joining your union
+
+Whether it’s employer/sector-specific questions, complex rate calculation rules, deduction-at-source, or Direct Debit, the Join Together platform adapts to meet all the specific joining needs of your union.
+
+## Seamless integration with your existing CRM and processes
+
+Join Together lives in harmony alongside your existing membership system, and is able to automatically feed new applications directly into your CRM ready for your membership team to process. Including Salesforce, Dynamics365, APT Stratum, Wild Apricot and more.
+
+## Works alongside your website
+
+The Join Together system can be fully customised to fit with your existing brand and website, guaranteeing a smooth user experience for visitors joining your union. From your logo to brand colours, fonts and more.
+
+## Built to work beautifully on mobile and desktop
+
+With as much as 50% of applicants using a mobile device, you really cannot afford to offer a sub-standard experience to mobile users. Join Together’s forms work perfectly across all devices.
+
+## Accessible and inclusive by default
+
+Our forms are designed to meet the W3C Web Accessibility guidelines, ensuring everyone – no matter their particular access needs – is able to join your union online.
+
+## Data ethics & security as standard
+
+Designed from the ground up to minimise the spread and retention of personal data, Join Together protects your data. Automatic encryption and bank grade security keeps application data safe in a secure, access-managed location. [Learn more about our security](/information-security).
+
+## A fully hosted and maintained service
+
+We support and maintain the Join Together platform so you don't have to, ensuring it's always up-to-date, secure, and available when your members need it.
```

Comment:
```suggestion
We support and maintain the Join Together platform so you don't have to, ensuring it's always up to date, secure, and available when your members need it.
```

No need to hyphenate when not a compound adjective. You could use it to say "an up-to-date application", but it doesn't need to be a compound word when used without a noun.

Yes, hyphenation is a personal hobby horse. I find their use and etymology fascinating – especially their removal due to common usage. "To-day" used to be hyphenated, as did "sub-standard"...